tests/test_users/test_schemas.py

```diff
@@ -1,28 +1,23 @@
 import copy
 from typing import Any
-from datetime import date
+from datetime import datetime, timedelta
 
 import pytest
 from pydantic.error_wrappers import ValidationError
 
 from src.apps.user.schemas import UserRegisterSchema
+from src.core.factories import UserFactory
 
-
-
-def test_passwords_are_not_identical(
-    register_data: dict[str, Any]
-):
-    user_data = copy.copy(register_data)
-    user_data['password_repeat'] = user_data['password'] + "abcde"
-    with pytest.raises(ValidationError) as exc:
+@pytest.mark.parametrize('incorrect_password', 'passwordX')
```

Comment:
Jezeli uzywasz jednego parametru to ta parametryzacja nic nie daje i nie rozni sie to od zwyklego testu. Z reguly parametrize uzywasz, kiedy chcesz dany test odpalic wiele razy na roznych danych, tu jest tylko jedna dana.
Mozesz np dac dwa hasla, jedno spoko drugie nie i sparametryzowac zeby w jednym przypadku oczekiwalo wyjatku, a w drugim nie. W googlu powinno sie dac znalezc jak to ogarnac